doctor/cores/api_ingress.py

```
@@ -0,0 +1,355 @@
+"""
+هسته ورودی API اپ Doctor
+Doctor App API Ingress Core
+"""
+
+from rest_framework.decorators import api_view, permission_classes
```

Ruff (0.12.2)

`rest_framework.decorators.api_view` imported but unused

Remove unused import

(F401)